src/types.db

```diff
@@ -78,6 +78,9 @@ frequency_offset	value:GAUGE:-1000000:1000000
 frequency		value:GAUGE:0:U
 fscache_stat		value:DERIVE:0:U
 gauge			value:GAUGE:U:U
+gps_sat			value:GAUGE:0:25
```

Comment:
Please rename this type to `satellites` and remove the upper limit (i.e. set it to `U`).